x/deployment/types/msgs.go

```diff
@@ -64,7 +67,7 @@ func (msg MsgUpdateDeployment) ValidateBasic() error {
 		return err
 	}
 
-	if err := sdk.VerifyAddressFormat(msg.Version); err != nil {
+	if len(msg.Version) == 0 {
```

Comment:
this should be 224/8 bytes right?   maybe define a constant somewhere, maybe defined from the package that the hash method comes from.

Reply:
I modified this to match the length of the version, it lead down a rabbit hole. Thankfully saved by the integration tests.

There is some place where this check runs and it seems to silently stop the update.  Created an issue to tackle this when it's more important: https://github.com/ovrclk/akash/issues/749